src/storage.h

```diff
@@ -46,6 +46,11 @@ namespace cycamore {
 /// Normal (optional)
 /// active_buying_mean is the mean length of the active buying period if active_buying_frequency_type is Normal
 /// active_buying_std is the standard deviation of the active buying period if active_buying_frequency_type is Normal
+/// active_buying_end_probability is the probability that at any given timestep, the agent ends the active buying period if
+/// the active buying frequency type is Binomial
+/// active_buying_disruption_probability is the probability that in any given cycle, the agent undergoes a disruption (disrupted 
+/// active period) if the active buying frequency type is FixedWithDisruption
+/// active_buying_disruption is the length of the disrupted active cycle if the active buying frequency type is FixedWithDisruption
```

Comment:
As we add so many options here, I wonder if we can add some formatting (not sure what it does to Doxygen?).  Here's an example, but it would be useful throughout.
```suggestion
/// active_buying_end_probability is the probability that at any given timestep, the agent ends the active buying 
///                               period if the active buying frequency type is Binomial
/// active_buying_disruption_probability is the probability that in any given cycle, the agent undergoes a disruption 
///                               (disrupted active period) if the active buying frequency type is FixedWithDisruption
/// active_buying_disruption is the length of the disrupted active cycle if the active buying frequency type is 
///                               FixedWithDisruption
```